build.gradle.kts

```diff
@@ -61,24 +67,24 @@ repositories {
     jcenter()
 }
 
-fun DependencyHandlerScope.includeAndExpose(dep: String) {
+fun DependencyHandlerScope.includeAndExpose(dep: Any) {
     modApi(dep)
     include(dep)
 }
 
 dependencies {
-    minecraft(group = "com.mojang", name = "minecraft", version = Minecraft.version)
-    mappings(group = "net.fabricmc", name = "yarn", version = Fabric.YarnMappings.version, classifier = "v2")
-
-    modImplementation(group = "net.fabricmc", name = "fabric-loader", version = Fabric.Loader.version)
-
-    includeAndExpose(Jetbrains.Kotlin.stdLib)
-    includeAndExpose(Jetbrains.Kotlin.stdLibJkd8)
-    includeAndExpose(Jetbrains.Kotlin.stdLibJkd7)
-    includeAndExpose(Jetbrains.Kotlin.reflect)
-    includeAndExpose(Jetbrains.annotations)
-    includeAndExpose(Jetbrains.KotlinX.coroutinesCore)
-    includeAndExpose(Jetbrains.KotlinX.coroutinesJdk8)
+    minecraft(group = "com.mojang", name = "minecraft", version = minecraftVersion)
+    mappings(group = "net.fabricmc", name = "yarn", version = minecraftVersion+"+build.1", classifier = "v2")
+
+    modImplementation("net.fabricmc:fabric-loader:_")
```

Comment:
what is this `_` stuff?

Reply:
_ gets replaced with what is in `versions.properties` by a plugin loaded in settings.gradle.kts

Reply:
see https://github.com/jmfayard/refreshVersions

Reply:
and it got removed because it does nto play well with something in there .. (i assume loom) in some environments